stacks/dbrs/redis/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM clearlinux AS build-redis
+MAINTAINER otc-swstacks@intel.com
+
+RUN swupd bundle-add --quiet --no-progress git c-basic devpkg-ndctl os-testsuite-phoronix-server
```

Comment:
os-testsuite-phoronix-server seems odd to be adding. Why are is it being pulled in?

Reply:
We require libnuma, this is the lowest size bundle containing it.

Reply:
@aduejazz is there any other package required from os-testsuite-phoronix?

Reply:
unfortunately we only need libnuma.so, and iirc that's the smallest bundle containing that library.

Reply:
Ah! I created a numa devpkg that'll show up next week.

Reply:
thanks, really appreciated!